_people/gaBBar.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: page
+title: Gaurav Sinha
+name: gaBBar
+position: Member
+---
+<img alt="gaBBar" width="250px" src= "//Users/gabbar/Downloads/gaBBar.jpg" />
```

Comment:
Sorry for the late comment. But this src won't go anywhere! You might need to add your image or point it to a gravatar!